core/types/block.libevm_test.go

```diff
@@ -135,6 +142,22 @@ func TestHeaderHooks(t *testing.T) {
 		assert.Equalf(t, &stub.setHeaderToOnUnmarshalOrDecode, hdr, "%T after RLP decoding with hook", hdr)
 	})
 
+	t.Run("PostCopy", func(t *testing.T) {
+		hdr := new(Header)
+		stub := &stubHeaderHooks{
+			accessor: extras.Header,
+			copied: &stubHeaderHooks{
+				suffix: []byte("copied"),
+			},
+		}
+		extras.Header.Set(hdr, stub)
+		cpy := CopyHeader(hdr)
+
+		copiedStub := extras.Header.Get(cpy)
+
+		assert.Equal(t, stub.copied, copiedStub)
```

Comment:
```suggestion
		hdr := new(Header)
		stub := &stubHeaderHooks{
			accessor: extras.Header,
			toCopy: &stubHeaderHooks{
				suffix: []byte("copied"),
			},
		}
		extras.Header.Set(hdr, stub)

		got := extras.Header.Get(CopyHeader(hdr))
		assert.Equal(t, stub.toCopy, got)
```

My suggested changes aren't as broad as the GH diff suggests:

1. Changed `copied` to `toCopy`, in keeping with my earlier suggestion;
2. Spacing change to logically group operations as setup then assertion; and
3. Usage of "got" variable to signal that it's the value under test.